j2me/core/src/org/javarosa/j2me/storage/rms/RMSStorageUtility.java

```diff
@@ -248,7 +246,7 @@ public void write (Persistable p) throws StorageFullException {
             if (!setReserveBuffer(bytesNeededEstimate)) {
                 setClean();
                 //can't reserve needed space to update index
-                throw new StorageFullException();
+                throw new RuntimeException();
```

Comment:
Is that true for everywhere that we're replacing StorageFullException with RuntimeException? Just seems like if we're going to replace StorageFullExceptions with something more generic, it can't hurt to give some slightly descriptive message.